src/index.html

```diff
@@ -16,17 +16,33 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-bar__page">
+    <form
+      class="search-bar__form search-bar__form--big"
+      action="http://localhost:8080/"
```

Comment:
Don't use spaces in links. The action URL 'http://localhost:8080/' should not contain a trailing slash if not necessary.